pkgs/development/python-modules/parselmouth/default.nix

```
@@ -0,0 +1,71 @@
+{
+  lib,
+  buildPythonPackage,
+  cmake,
+  fetchFromGitHub,
+  future,
+  numpy,
+  pytest-lazy-fixture,
+  pytestCheckHook,
+  pythonOlder,
+  scikit-build,
+  setuptools,
+  wheel,
+}:
+
+buildPythonPackage rec {
+  pname = "parselmouth";
+  version = "0.4.4";
+  pyproject = true;
+  build-system = [
+    cmake
+    scikit-build
+    setuptools
+    wheel
+  ];
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "YannickJadoul";
+    repo = "Parselmouth";
+    # https://github.com/YannickJadoul/Parselmouth/issues/130
+    rev = "c2cbecc0ce4a0b5d3052cc6c6fbddf4bf133655d";
+    hash = "sha256-n6xc3kIfxLAwHG4N/C3i5wAcdb4kI015GB0QVh/zuVk=";
+  };
+
+  fmt = fetchFromGitHub {
+    owner = "fmtlib";
+    repo = "fmt";
+    rev = "7bdf0628b1276379886c7f6dda2cef2b3b374f0b";
+    sha256 = "sha256-SbzWk4u1Dt/944N+hiAu/zd49dUXTR2BXs2Ki8fZHiI=";
+  };
+
+  postPatch = ''
+    cp -r --no-preserve=all ${fmt.outPath}/* ./extern/fmt
+  '';
+
+  configurePhase = ''
+    export MAKEFLAGS=-j$NIX_BUILD_CORES
+  '';
+
+  dontUseCmakeConfigure = true;
+
+  dependencies = [ numpy ];
+
+  nativeCheckInputs = [
+    future
+    pytest-lazy-fixture
+    pytestCheckHook
+  ];
+
+  pythonImportsCheck = [ "parselmouth" ];
+
+  meta = {
+    description = "Praat in Python, the Pythonic way";
+    homepage = "https://github.com/YannickJadoul/Parselmouth";
+    changelog = "https://github.com/YannickJadoul/Parselmouth/releases/tag/v${version}";
+    license = lib.licenses.mit;
```

```suggestion
    license = lib.licenses.gpl3Plus;
```
https://github.com/YannickJadoul/Parselmouth/blob/c2cbecc0ce4a0b5d3052cc6c6fbddf4bf133655d/setup.py#L1-L16